boards/arm/google_twinkie_v2/board.cmake

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(dfu-util "--pid=0483:df11" "--alt=0" "--dfuse")
```

Comment:
Could you keep the `board_runner_args` at the top and `include` at the bottom? Not sure why it's done that way for the other boards but it seems like that's the case for pretty much all of them, let's keep it coherent unless we have a good reason not to.